src/QSymbolicsBase/basic_ops_homogeneous.jl

```diff
@@ -233,7 +232,7 @@ julia> anticommutator(A, B)
     op1
     op2
     function SAnticommutator(o1, o2) 
-        coeff, cleanterms = prefactorscalings([o1 o2], scalar=true)
+        coeff, cleanterms = prefactorscalings([o1 o2])
```

Comment:
same question about dropping the custom constructor and moving this to `anticommutator`